packages/react-ui/src/components/skeleton/index.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import delve from 'dlv'
+import PropTypes from 'prop-types'
+import { Element } from '../../primitives'
+import { styles } from './skeleton.styles'
+import { merge } from '../../utils'
+
+const getLinearBackground = theme => {
+  /** Read special keys from theme but make sure
+   * it doesn't break if the keys aren't present
+   * */
+  const backgroundColor =
+    delve(theme.components, 'Skeleton.backgroundColor') || ''
```

Comment:
Looks like `dlv` accepts a third argument as the default value (`delve(object, keypath, [default])`), can we use it here? Or you think the `||` is easier to understand for someone that doesn't know about this utility?

```suggestion
    delve(theme.components, 'Skeleton.backgroundColor', '')
```

Reply:
oh that's beautiful. I didn't realise